src/components/ScenarioParameters/ScenarioParameters.js

```diff
@@ -76,6 +76,9 @@ const ScenarioParameters = ({
     const initialStockParameter = currentScenario.data?.parametersValues?.find(el => el.parameterId === INITIAL_STOCK_PARAM_ID);
     setInitialStockDatasetId(initialStockParameter?.value);
     /// //////////////////////////////////////////////////////////////////////// INITIAL STOCK
+
+    // Reset parameters
+    resetParameters();
```

Comment:
This change is not a part of your branch, could you rebase your branch on **main** please ? :)

Reply:
Resovled by rebase on main